### ISSUE TITLE
make https usable for torrent

### ISSUE THREAD

**Issue by CaptSonic** [PCJones/UmlautAdaptarr#38]
I use a German private tracker, which I can only integrate via the templates in Prowlarr. Unfortunately, I can only select the https URL of the tracker there.
Is there a workaround for this?

**Comment by PCJones** [comment]
No, since a core principle of https is that it is encrypted there is no workaround.
Most trackers should offer a torznab API which will work fine with UmlautAdaptarr (you may have to select "Generic Torznab" instead of the predefined template).

I will create a workaround for TS in the somewhat near future as it's the largest German private tracker, however I won't do it for e.g. BTF (as I'm not on there)

**Comment by CaptSonic** [comment]
TS is the tracker I mean ;). Unfortunately I can't add it as generic.

**Comment by PCJones** [comment]
Then all you need is patience :D it will come eventually.

**Comment by xDaveN** [comment]
Thanks for your work so far, unfortunately I'm in a similar boat, plus I'm not on TS (yet).
I know you said you wont bother with other trackers, but I'd still love to see support for a couple of the higher tier German trackers. (:
I'm basically using predefined Prowlarr profiles for all of them, so I can't edit the URL and using a generic template would have me lose some functionality. :/

**Comment by raphaelM-sudo** [comment]
I did the following, and it seems to work:
Go to settings/applications in Prowlarr.
Click on the radarr/sonarr app to edit the settings.
Change "Prowlarr Server" to: "http://localhost:5005/_/<insert prowlarr domain here>".
localhost:5005 is the domain:port of UmlautAdaptarr in my case.

**Comment by PCJones** [comment]
> I did the following, and it seems to work: Go to settings/applications in Prowlarr. Click on the radarr/sonarr app to edit the settings. Change "Prowlarr Server" to: "http://localhost:5005/_/ [insert prowlarr domain here]". localhost:5005 is the domain:port of UmlautAdaptarr in my case.

Huh! Good catch, this might actually work

**Comment by raphaelM-sudo** [comment]
I am using Nginx Proxxy Manager for all my containers, so yes. It might work without it, if you configure Prowlarr on port 80 and 443 with SSL.
